### DISCUSSION TITLE
Japanese script sorting

### DISCUSSION THREAD

**Discussion by lokn84** [VoxelCubes/PanelCleaner#111] [Ideas]
Hello, i think it would be great if after the ocr process, there is a way to sort the ocr text box in reading order so the dialogues are easier to translate and typeset using typesetting plugin. Either manually after the ocr is done, or semi automatic with some manual adjustment, then export to the text files ready for chatgpt translation and typesetting in reading order.

**Comment by VoxelCubes** [comment]
Well it's your lucky day!
This is the norm since version 2.7, I recommend updating your panel cleaner.
